lib/cosi/cosi.go

```diff
@@ -104,6 +104,14 @@ type Signature struct {
 	Response  abstract.Secret
 }
 
+// NewSignature returns a pre-initialized signature with Secret.One
+func NewSignature(s abstract.Suite) *Signature {
```

Comment:
To marshal or unmarshal ? 
* To marshal a nil-signature -> you should not do that and it should throw an error if you try to do that. If you marshal  a `NewSignature()`, it just hides a bug.
* To unmarshal, the network library does it for you with `UnmarshalRegisteredType`.

More generally, a signature should be available only when the `Cosi` struct is ready to deliver one or delivered by the network. I don't really see the point of having a *default signature* (signature of what ? with what cosigners ? etc), but maybe you used it somewhere where it was needed ?  A simple `Ctrl-F` search in the PR did not show anything.